runtime/compiler/env/VMJ9Server.hpp

```diff
@@ -160,6 +160,8 @@ class TR_J9ServerVM: public TR_J9VM
    virtual uintptrj_t getFieldOffset(TR::Compilation * comp, TR::SymbolReference* classRef, TR::SymbolReference* fieldRef) override { return 0; } // safe answer
    virtual bool instanceOfOrCheckCast(J9Class *instanceClass, J9Class* castClass) override;
    virtual bool transformJlrMethodInvoke(J9Method *callerMethod, J9Class *callerClass) override;
+   using TR_J9VM :: isAnonymousClass;
```

Comment:
What is the purpose of this `using` statement?

Reply:
Once a name match is found in the a class scope, it doesn't look further up the inheritance tree for overloads. By specifying the 'using' declaration, you bring all of the overloads of 'isAnonymousClass' from 'TR_J9VM' into the scope of 'TR_J9ServerVM'. Then overloading works properly.

Reply:
Otherwise we have to override all the function of the same name.

Reply:
> Then overloading works properly.

Are you saying that without `using TR_J9VM :: isAnonymousClass;` the code does not work correctly?

Reply:
It does not compile. The definition of the virtual bool isAnonymousClass(TR_OpaqueClassBlock *j9clazz) will be visible but the virtual bool isAnonymousClass(J9ROMClass *romClass) will not be visible.